server.js

```diff
@@ -58,6 +58,7 @@ app.use(requestLanguage({
   languages: Object.keys(i18n.locales),
   cookie: {
     name: 'language',
+    options: {maxAge: 24 * 60 * 60 * 1000},
```

Comment:
What do you say to bumping this to a month?

```suggestion
    options: {maxAge: 30 * 24 * 60 * 60 * 1000},
```

Reply:
sounds good to me!